From a35242420501c489888ed0b8a594104dae410b83 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Mon, 3 Dec 2007 20:03:15 +0200
Subject: [PATCH] Makefile.in: rewrite install target

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile.in |   42 ++++++++++++++++++++++++++++++++----------
 1 files changed, 32 insertions(+), 10 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 3785448..68c9ad8 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -1,30 +1,52 @@
 #wput makefile
-PACKAGE     = @PACKAGE@
-prefix      = @prefix@
-datadir     = $(prefix)/share
-mandir	    = $(datadir)/man/man1
-exec_prefix = @exec_prefix@
-bindir=@bindir@
+# PACKAGE     = @PACKAGE@
+# prefix      = @prefix@
+# datadir     = $(prefix)/share
+# mandir	    = $(datadir)/man/man1
+# exec_prefix = @exec_prefix@
+# bindir=@bindir@
+
+DESTDIR =
+prefix          = /usr
+exec_prefix     = $(prefix)
+man_prefix      = $(prefix)/share
+
+INSTALL = /usr/bin/install
+INSTALL_BIN     = $(INSTALL) -m 755
+INSTALL_DATA    = $(INSTALL) -m 644
+INSTALL_OBJS    = wput
+
+MANDIR          = $(DESTDIR)$(man_prefix)/man/man1
+BINDIR          = $(DESTDIR)$(exec_prefix)/bin
 
 all clean:
 	cd po && $(MAKE) $(MAKEDEFS) $@
 	cd src && $(MAKE) $(MAKEDEFS) $@
-	
+
 win-clean:
 	cd src && $(MAKE) $(MAKEDEFS) $@
 
-install: all
+install-old: all
 	cd po && $(MAKE) $(MAKEDEFS) $@
 	install -m0755 wput $(bindir)
 	install -m0644 doc/wput.1.gz $(mandir)
 	@echo "----------------"
 	@echo "Wput installed. See 'wput -h' or 'man wput' for usage information."
 	@echo "Further documentation is located in the doc/USAGE.* files."
-	@echo 
+	@echo
 	@echo "Wput is not perfect, so please report any bugs you notice (see BUGS-section"
 	@echo "in the manpage)."
 	@echo "----------------"
 uninstall:
 	rm -f $(bindir)/wput
 	rm -f $(mandir)/wput.1.gz
-	
+
+install-bin:
+	$(INSTALL_BIN) -d $(BINDIR)
+	$(INSTALL_BIN) -s $(INSTALL_OBJS) $(BINDIR)
+
+install-man:
+	$(INSTALL_BIN) -d $(MANDIR)
+	$(INSTALL_DATA) doc/*.1* $(MANDIR)
+
+install: install-bin install-man
-- 
1.5.3.GIT

