From: Jari Aalto <jari.aalto@cante.net>
Date: Tue, 15 Dec 2009 23:35:33 +0200
Subject: [PATCH] src/Makefile.in: add CFLAGS_EXTRA

--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -5,7 +5,7 @@
 bindir=@bindir@
 localedir=$(prefix)/share/locale
 CC=@CC@
-CFLAGS= @CFLAGS@ -DLOCALEDIR=\"$(localedir)\" @GNUTLS_CFLAGS@
+CFLAGS= @CFLAGS@ -DLOCALEDIR=\"$(localedir)\" @GNUTLS_CFLAGS@ $(CFLAGS_EXTRA)
 LIBS=@LIBINTL@ @LIBS@ @GNUTLS_LIBS@
 EXE=../wput
 GETOPT=@GETOPT@
--- a/Makefile.in
+++ b/Makefile.in
@@ -17,11 +17,11 @@
 
 install: all
 	cd po && $(MAKE) $(MAKEDEFS) $@
-	mkdir -p $(destdir)$(bindir) 
-	mkdir -p $(destdir)$(mandir) 
+	install -m0755 -d $(destdir)$(bindir)
+	install -m0755 -d $(destdir)$(mandir)
 	install -m0755 wput $(destdir)$(bindir)
 	install -m0644 doc/wput.1.gz $(destdir)$(mandir)
-	ln -fs $(destdir)$(bindir)/wput $(destdir)$(bindir)/wdel
+	cd $(destdir)$(bindir) && ln -s wput wdel
 	install -m0644 doc/wdel.1.gz $(destdir)$(mandir)
 	@echo "----------------"
 	@echo "Wput and Wdel installed. See 'wput/wdel -h' or 'man wput/wdel' for"
@@ -36,4 +36,4 @@
 	rm -f $(destdir)$(bindir)/wdel
 	rm -f $(destdir)$(mandir)/wput.1.gz
 	rm -f $(destdir)$(mandir)/wdel.1.gz
-	
+
